deploy/crds/csi.ibm.com_v1_ibmblockcsi_cr.yaml

```diff
@@ -57,7 +57,7 @@ spec:
     imagePullPolicy: IfNotPresent
   - name: csi-provisioner
     repository: k8s.gcr.io/sig-storage/csi-provisioner
-    tag: "v2.1.0"
+    tag: "v2.2.0"
```

Comment:
what about snapshotter `v3.0.3`?

Reply:
I thought we don't want to use beta versions, but done.